OrbitLinuxTracing/TracerThread.h

```diff
@@ -7,12 +7,23 @@
 
 #include <atomic>
 #include <memory>
+#include <mutex>
 #include <optional>
 #include <regex>
 #include <vector>
 
+#include "PerfEvent.h"
+#include "absl/container/flat_hash_map.h"
+
+struct perf_event_header;
+class Function;
```

Comment:
Apart from the fact that the header should be included instead, isn't this in the LinuxTracing namespace?